src/locales/ko/trainer-names.json

```
@@ -173,5 +173,5 @@
   "vicky": "영아",
   "vito": "철준",
   "bug_type_superfan": "벌레타입 마니아",
-  "expert_pokemon_breeder": "Expert Pokémon Breeder"
+  "expert_pokemon_breeder": "전문 포켓몬 브리더"
```

```suggestion
  "expert_pokemon_breeder": "포켓몬 전문 브리더"
```
앞뒤 단어를 바꾼 건데 
국내는 
```말티즈 전문 브리더``` 나  ```고양이 전문 브리더``` 라는 표현을 써서
그 어순이 더 익숙하지 않을까 싶어서 적어뒀어영

---

무방할 것 같습니다